source/Plugins/SymbolFile/DWARF/HashedNameToDIE.h

```diff
@@ -53,16 +53,24 @@ class DWARFMappedHash {
   };
 
   struct DIEInfo {
-    DIERef die_ref;
-    dw_tag_t tag;
-    uint32_t type_flags;          // Any flags for this DIEInfo
-    uint32_t qualified_name_hash; // A 32 bit hash of the fully qualified name
-    uint64_t strp;                // 64 bit string table offset
+    dw_offset_t die_offset = DW_INVALID_OFFSET;
+    dw_tag_t tag = 0;
 
-    DIEInfo();
-    DIEInfo(dw_offset_t c, dw_offset_t o, dw_tag_t t, uint32_t f, uint32_t h);
+    /// Any flags for this DIEInfo
+    uint32_t type_flags = 0;
 
-    explicit operator DIERef() const { return die_ref; }
+    /// A 32 bit hash of the fully qualified name
+    uint32_t qualified_name_hash = 0;
+
+    // 64 bit string table offset
+    uint64_t strp;
```

Comment:
`strp` is not upstream. It's also unused in swift-lldb as far as I understand. It was added with the initial "Add Swift support to LLDB" https://github.com/apple/swift-lldb/commit/84ef5bf74617

Upstream code was touched in https://reviews.llvm.org/D63399. It mentions a series of patches to come.

Reply:
So we should just remove it then?

Reply:
Well, it looks like we lost functionality here, but as I don't know what it does, I cannot say whether or not we need it. I only see a unreasonable merge conflict.